lntest/itest/lnd_test.go

```diff
@@ -267,6 +267,13 @@ func closeChannelAndAssert(ctx context.Context, t *harnessTest,
 	net *lntest.NetworkHarness, node *lntest.HarnessNode,
 	fundingChanPoint *lnrpc.ChannelPoint, force bool) *chainhash.Hash {
 
+	return closeChannelAndAssertType(ctx, t, net, node, fundingChanPoint, false, force)
+}
+
+func closeChannelAndAssertType(ctx context.Context, t *harnessTest,
+	net *lntest.NetworkHarness, node *lntest.HarnessNode,
+	fundingChanPoint *lnrpc.ChannelPoint, anchors, force bool) *chainhash.Hash {
```

Comment:
can just take commit type instead of the bool?

Reply:
yes, so the reason that I made it not the commit type, is that this function is also used for coop closes when no anchor is expected (even though the chan type is still anchors).

Reply:
Oh, because you don't have access to the `force` bool in `assertChannelClosed`?

Reply:
Yes that. I could add it probably. But it would also mean that all calls to these assertions need the commit type. Now I can get away by passing in `false` for `anchors` for all the existing tests, while when a type is expected, I would want to make sure that the actual channel type is passed in (legacy or tweakless).